doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -114,13 +141,23 @@ Declaring Coercions
 -------------------------
 
 .. cmd:: Coercion @reference : @class >-> @class
-         Coercion @ident {? @univ_decl } @def_body
+         Coercion @ident_decl @def_body
+         Coercion @qualid
 
   The first form declares the construction denoted by :token:`reference` as a coercion between
-  the two given classes.  The second form defines :token:`ident`
-  just like :cmd:`Definition` :n:`@ident {? @univ_decl } @def_body`
-  and then declares :token:`ident` as a coercion between it source and its target.
+  the two given classes.  The second form defines :token:`ident_decl`
+  just like :cmd:`Definition` :n:`@ident_decl @def_body`
+  and then declares :token:`ident_decl` as a coercion between it source and its target.
   Both forms support the :attr:`local` attribute, which makes the coercion local to the current section.
+  The third form enables to modify the :attr:`reversible` attribute of
+  the already declared coercion :token:`qualid`.
```

Comment:
```suggestion
  The third form enables modifying the :attr:`reversible` attribute of
  the already declared coercion :token:`qualid`.
```

I get the impression that French may always use the infinitive where English uses the gerund.  I see this kind of typo often :-(.